src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$month-length: 28;
```

Comment:
This needs to be renamed. `$min-month-length`.
Also, add `$max-month-length` with proper value and use it.